### ISSUE TITLE
gtceu gregtech-1.12.2-2.5.0-beta crush

### ISSUE THREAD

**Issue by ferreiraalex** [codetaylor/artisan-worktables#266]
https://mclo.gs/iROpks5
CRUSH

**Comment by Speecker** [comment]
Seems like the most recent AI version (v1.3.0) is incompatible with GTCEu (v2.5.3) (an unofficial fork with continued support): https://pastebin.com/cxGYntBM
Will there be an update that fixes compatibility with GTCEu ?

**Comment by codetaylor** [comment]
Yeah GTCEu changed their API again. I'll update when I get to it.

**Comment by DrDarkfire** [comment]
Hi, just writing for a gentle reminder for this issue.

**Comment by codetaylor** [comment]
Yeah, I looked into this some time ago and wasn't able to find a clean way to make it work with the changed API.

**Comment by DrDarkfire** [comment]
I was reading a bit of the GT source, and I came across [ToolHelper.java](https://github.com/GregTechCEu/GregTech/blob/master/src/main/java/gregtech/api/items/toolitem/ToolHelper.java). Would this help? I found the damageItem method in there.

**Comment by codetaylor** [comment]
No, it's more complicated than that. Thank you though. :)
